player.js

```diff
@@ -16,19 +16,40 @@
 
 
 
-
 const Mplayer = require('mplayer');
 const fs = require('fs');
+const argv = require('minimist')(process.argv.slice(2));
+
+if (argv.h || argv.help) {
+	console.log('usage: node player.js [options]\n')
+	console.log('  -p [PORT_ID]    sets the port used by the server')
+	console.log('  -novideo        play only audio')
+	console.log('  -v              display the path of played video')
+	console.log('  -debug          set verbose and debug to true for mplayer')
+	console.log('  -h              display this help')
```

Comment:
Isn't the usual convention to use 
`-h` (single dash, single letters)
`--help` (double dashes, multiple letters)
?

(Also, it should be `-h -help` here)